src/Symfony/Component/DependencyInjection/Tests/Loader/XmlFileLoaderTest.php

```diff
@@ -1318,7 +1318,7 @@ public function testInvalidBinaryKeyType()
         $loader = new XmlFileLoader($container, new FileLocator(self::$fixturesPath.'/xml'));
 
         $this->expectException(InvalidArgumentException::class);
-        $this->expectExceptionMessage(\sprintf('Tag "<property>" with key-type="binary" does not have a valid base64 encoded key in "%s".', self::$fixturesPath.'/xml/key_type_incorrect_bin.xml'));
+        $this->expectExceptionMessageMatches('/^Tag "<property>" with key-type="binary" does not have a valid base64 encoded key in "(.+)key_type_incorrect_bin.xml".$/');
```

Comment:
```suggestion
        $this->expectExceptionMessage(\sprintf('Tag "<property>" with key-type="binary" does not have a valid base64 encoded key in "%s/xml%skey_type_incorrect_bin.xml".', self::$fixturesPath, \DIRECTORY_SEPARATOR));
```

I think that's what do in other tests

Reply:
Alright :+1: